src/script/sign.h

```diff
@@ -103,7 +103,7 @@ std::vector<unsigned char> signmessage(const BYTEARRAY &data, const CKey &key)
     return vchSig;
 }
 
-/** sign arbitrary data using the same algorithm as the signmessage/verifymessage RPCs and OP_DATASIGVERIFY */
+/** sign arbitrary data using the same algorithm as the signmessage/verifymessage RPCs and OP_CHECKDATASIG(VERIFY) */
```

Comment:
its CDS doesn't use the pubkey recoverable signature does it?  so AFAIK we need to delete "...and OP_CHECKDATASIG(VERIFY)"